package.json

```diff
@@ -27,14 +34,18 @@
     "ajv-formats": "2.1.1",
     "ava": "4.2.0",
     "debug": "4.3.4",
+    "dotenv": "16.0.1",
     "eslint": "8.17.0",
     "eslint-config-prettier": "8.5.0",
     "husky": "7.0.4",
     "lint-staged": "12.3.8",
     "prettier": "2.6.2"
   },
   "lint-staged": {
-    "*.{mjs,js,css,md}": ["npm run prettier", "npm run eslint"]
+    "*.{mjs,js,css,md}": [
```

Comment:
I won't block the PR because of that, but for some reason you and I are always reformatting JSON files.